lesson18/index.js

```diff
@@ -88,6 +109,59 @@ const controlBtnBehavior = () => {
   nextBtn.classList.remove("is-disabled");
 }
 
+const setPagenation = (slides) => {
+  const slider = document.getElementById("js-slider");
+  const pagenation = document.createElement("div");
+  pagenation.classList.add("pagenation");
+  const pagenationWrapper = document.createElement("div");
+  pagenationWrapper.classList.add("pagenationWrapper");
+  const slideLength = slides.length;
+
+  slides.map((slide,index) => {
+    const span = document.createElement("span");
+    span.classList.add("pagenationBullet");
+    index === 0 ? span.classList.add("is-active") : span.classList.remove("is-active");
+    span.setAttribute("data-slideNo",`slideNo${index}`);
```

Comment:
Q. Is this attribute value required? I don't think it is used elsewhere.

Reply:
[eb609b9](https://github.com/hippopo-moo/js-handson-work/pull/20/commits/eb609b991927af3535166aa35986ecae3b75b675)

True... I put it on with the intention of using it and forgot to turn it off;;
 I deleted it because it was unnecessary.